### DISCUSSION TITLE
New channels

### DISCUSSION THREAD

**Discussion by bonridley** [iptv-org/community#1023] [Show and tell]
BD Network (Argentina; Not 24/7) broadcasting at 8PM (EST) to 2AM (EST)
https://cablemor.andremor.dev/live/bdnetwork/index.m3u8
Logo: 
![imagen](https://github.com/iptv-org/iptv-org.github.io/assets/157573538/832fe362-4268-451a-8e39-48d0b95588f4)

Teleposadas (Argentina)
https://5e06e5e8c2e27.streamlock.net:1936/videonow/teleposadas12/playlist.m3u8
Logo: 
![imagen](https://github.com/iptv-org/iptv-org.github.io/assets/157573538/98483d17-97a5-4789-a9f9-274ddb3de757)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#how-to-add-a-new-stream-link-to-a-playlists
